### DISCUSSION TITLE
Stretch bar to notch

### DISCUSSION THREAD

**Discussion by kristiansordal** [Jean-Tinland/simple-bar#378] [Q&A] [answered]
I have a MBP 14" with the notch in the middle. I really like this menubar, but I have one issue which I hope to get help with.

<img width="1512" alt="Screenshot 2022-07-19 at 18 00 39" src="https://user-images.githubusercontent.com/94058487/179809746-4d892ce5-fb20-4344-87bd-96b7eb16229f.png">

As seen in this image, the spaces / dekstops indicators doesn't fill the screen up to the notch (not seen in the image), in the same way the widgets on the right hand side does. 

How can I stretch or increase the length of the bars so that they fill up the screen and gets rid of the empty space?

Thanks!

**Comment by Jean-Tinland** [comment, marked as answer]
You can add these styles in the "Custom styles" tab in the settings module:

```css
.simple-bar--no-bar-background .spaces {
  min-width: 600px;
}
```

Then you can adjust the `min-width` value.

**Comment by kristiansordal** [comment]
Thanks! This works great, how can I right align the data part of the bar?
